### PR TITLE
fish9's First GoodieCreated my first Goodie

### PR DESCRIPTION
**Pull request by fish9**
# Goodie Pull Request Template

Using this template will help us better understand your Instant Answer and assist you when necessary.  Simply copy and paste the markdown below into the description of your GitHub pull request and complete as appropriate.

**What does your Instant Answer do?**
tells me how unawesome I am

**What problem does your Instant Answer solve (Why is it better than organic links)?**
finished quickstart tutorial

**What is the data source for your Instant Answer? (Provide a link if possible)**
noe

**Why did you choose this data source?**

**Are there any other alternative (better) data sources?**

**What are some example queries that trigger this Instant Answer?**
duckduckhack fish9

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
none

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**
no

**Which existing Instant Answers will this one supercede/overlap with?**

**Are you having any problems? Do you need our help with anything?**
I was having problems doing this in Chrome, Codio would not let me fork!  I got an email from Joel Moss from Codio saying there were problems forking with Chrome.  Redid the tutorial in firefox, worked fine.

**Where did you hear about DuckDuckHack? (For first time contributors)**
saw the posting on jobs.perl.org, decided to try the tutorial and get involved

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
just says fish9 is awesome

## Checklist
Please place an 'X' where appropriate.

```
- [X] Added metadata and attribution information (https://duck.co/duckduckhack/metadata)
- [X] Wrote test file and added to t/ directory (https://duck.co/duckduckhack/goodie_tests)
- [] Verified that instant answer adheres to design guidelines (https://duck.co/duckduckhack/design_styleguide)
- [] Verified that instant answer adheres to code styleguide (https://duck.co/duckduckhack/code_styleguide)
- [] Tested cross-browser compatibility

    Please let us know which browsers/devices you've tested on:
    - Windows 8
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] IE 10

    - Windows 7
        - [] Google Chrome
        - [X] Firefox
        - [] Opera
        - [] IE 8
        - [] IE 9
        - [] IE 10

    - Windows XP
        - [] IE 7
        - [] IE 8

    - Mac OSX
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] Safari

    - iOS (iPhone)
        - [] Safari Mobile
        - [] Google Chrome
        - [] Opera

    - iOS (iPad)
        - [] Safari Mobile
        - [] Google Chrome
        - [] Opera

    - Android
        - [] Firefox
        - [] Native Browser
        - [] Google Chrome
        - [] Opera

```

### CODE REVIEW

File: lib/DDG/Goodie/IsAwesome/fish9.pm [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+zci is_cached   => 1;
+
+name "IsAwesome fish9";
+description "My first Goodie, it lets the world know that fish9 is awesome";
+primary_example_queries "duckduckhack fish9";
+category "special";
+topics "special_interest", "geek";
+code_url "https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/lib/DDG/Goodie/IsAwesome/fish9.pm";
+attribution github => ["https://github.com/fish9", "fish9"],
+            twitter => "TwitterUserName";
```
Comment (moollaza) [inline lib/DDG/Goodie/IsAwesome/fish9.pm]: Care to provide your Twitter handle? If not, please remove this line

**Comment by moollaza** [comment]
@fish9 thanks a lot for contributing and congrats on your first Goodie! :cake: 

Just one small fix (twitter attribution) and this should be good for merge!

**Comment by fish9** [comment]
Hi Zaahir:
Thanks, I am not clear on how I checkout and checkin my perl module and make sure you get the latest "pushed" back to you.   Not sure what "Changes not staged for commit" means?Can you please give me a quick overview of how you checkout a file from git, mod, check back in and then make it available, do I need a new pull request?
Appreciate any explanation of this.
Thanks,Brent 

[04:56 PM codio@chicago-float zeroclickinfo-goodies {master}]$ git checkout                                                                          M       lib/DDG/Goodie/IsAwesome/fish9.pm                                                                                                            Your branch is up-to-date with 'origin/master'.                                                                                                      [04:56 PM codio@chicago-float zeroclickinfo-goodies {master}]$ git commit                                                                            # On branch master                                                                                                                                   # Your branch is up-to-date with 'origin/master'.                                                                                                    #                                                                                                                                                    # Changes not staged for commit:                                                                                                                     #       modified:   lib/DDG/Goodie/IsAwesome/fish9.pm                                                                                                #                                                                                                                                                    no changes added to commit                                                                                                                           [04:56 PM codio@chicago-float zeroclickinfo-goodies {master}]$ git help   
Date: Tue, 10 Feb 2015 08:41:16 -0800

**Comment by mintsoft** [comment]
@fish9 looking at it you've sorted out the issues you were having with git?

Congrats on your first goodie!

**Comment by moollaza** [comment]
@fish9 congrats! It's live: https://duckduckgo.com/?q=duckduckhack+fish9&ia=answer

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's & Issues or even submit more Instant Answers!

**Pull request merged**